### ISSUE TITLE
no access to api(?)

### ISSUE THREAD

**Issue by punkmatsche** [akiver/CSGO-Demos-Manager#623]
i have been using CSGO demo manager for a long time now, and only now have issues loading suspects, adding accounts using the profile url or id, or checking for Bans. I cant even sync User names and profile pictures, but i can add people to the account list from an analyzed match.
It looks to me like any Api request to Steam gets denied.

im not a programmer, nor am i versatile enough to check that, but the issue is still there after resetting windows and installing the csgo demo manager again, even a full harddrive wipe and bios reset didnt fix that, anything else works, analyzing demos.

If thats the case, is that something i have to fix on my end? do i have to contact Steam to get it fixed?
I been googling for that error for a few days now, trying to use my limited brain capacity to figure that out, but i cant get it to work please help

**Comment by angoisser** [comment]
Same issues for me, it's been happening for about a week now. Whenever I analyze demos it seems like it doesn't detect who is banned anymore. Few weeks prior to that, I had the issue where the suspects list didn't load at all. It was fixed for a little but now it sadly doesn't work anymore.

I haven't gone as far as resetting my PC or hard drive, but I did clear the cache from CSGO Demos Manager and reinstalled the program. Didn't fix anything either. I guess it is just broken for now.
